no-olm/migration/migration.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
Was this standalone script meant to be checked in? I see a comment that it is a PoC

Reply:
It is poc but its easier to test changes to the script in script form. I'll remove it before merging in